package.json

```diff
@@ -20,8 +20,9 @@
     "bundle-profile:analyze": "npm run build:bundle-profile && webpack-bundle-analyzer ./stats.json",
     "clean": "rimraf dist",
     "deploy-console-to-server": "cp -r ./dist/* ./server/infinispan-server/static/console",
-    "cy:run": "cypress run --headless",
-    "cy:e2e": "cypress open"
+    "cy:run": "cypress run --headless --spec 'cypress/e2e/!(xsite.cy.js)'",
+    "cy:e2e": "cypress open",
+    "cy:run-xsite": "cypress run --headless --spec 'cypress/e2e/xsite.cy.js'"
```

Comment:
The server `./run-xsite.tests.sh` stop after running, how can we then utilize this command?